src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -1294,7 +1291,7 @@ def forward(
 
         is_index_masked = attention_mask < 0
         is_index_global_attn = attention_mask > 0
-        is_global_attn = is_index_global_attn.flatten().any().item()
+        is_global_attn = torch.any(is_index_global_attn).item()  # the ONNX export should record is_global_attn == True
```

Comment:
Will try to review in more details later, but a question here:

Why we need to :
 - use `torch.any` instead of the tensor's `any()` method
 - remove `flatten`

Are these just some simplification rather than necessarity?

Furthermore, could you explain a bit to me (ONNX newbie) what it means 

``` 
# Record `is_global_attn == True` to enable ONNX export
```

Reply:
Right, it's probably unnecessary. I'll fix.

The comment means that in a normal case scenario, the control flows should be done having `is_global_attn == True`. This means that the example provided to `torch.onnx.export` should have at least at least one id of `global_attention_mask` non-zero.

Reply:
Thank you. I understand the comment better now.

One more question however, if the case `is_global_attn = True` is recorded and used few lines below (note here is an assignment, not `==`) during export time, would we have trouble during inference time if we feed an input without any positive global attention mask?

Reply:
Yes exactly. IMO the ONNX export is lacking in this sense: since we record only a single path, there may well be cases where the exported ONNX is invalid / will throw weird errors), while it is valid in other cases.

I was thinking to have some kind of support (in Optimum exporters) for an external file to the `.onnx` that would specify which cases are supported with the exported onnx.